spring-cloud-gcp-data-firestore/src/test/java/com/google/cloud/spring/data/firestore/FirestoreTemplateTests.java

```diff
@@ -245,31 +278,31 @@ void deleteByIdTest() {
 
   private void mockCommitMethod() {
     doAnswer(
-            invocation -> {
```

Comment:
Could you help remove the formatting changes introduced in this PR that are outside of the tests edited/added? (For consistency with the rest of the repository, which conforms to [Google Java Style](https://google.github.io/styleguide/javaguide.html))

Reply:
Done @emmileaf , however I am finding that if I use the Google Intellij Style xml available here - https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml, then it tends to reformat the code the way I had originally submitted. Is it possible to get hold of a intellij style that conforms to the style here.

Reply:
Because we've moved from Spring style to Google style over time, it's best to avoid reformatting a whole file as part of a feature PR. Try selecting a block of code you are working on and formatting just the block instead. Sorry about that!

Reply:
Ah, okay, that makes sense @elefeint. I will ensure that only my changes are made with the Google style.